### ISSUE TITLE
loss doesn't decrease

### ISSUE THREAD

**Issue by shukaizheng** [jjery2243542/adaptive_voice_conversion#11]
Is it something wrong？The loss_rec=0.25 and loss_kl = 0.28, when training step is 40 000, and it's doesn't decrease any more. The training step is 100 000 now, but the loss_rec is still keep 0.25 and loss_kl 0.28. Is it normal?（lambda keep 1 when training step=20 000）

**Comment by jjery2243542** [comment]
Hi,
although loss only slightly decreases, the model is becoming better in terms of generation.

**Comment by youngsuenXMLY** [comment]
@wotulong what is configuration for your training? I ran the same code, but I can't reproduce the loss_rec and loss_kl as small as yours. My loss_rec stayed around 0.33 and loss_kl stayed around 0.3.
Did you make any optimization on the training process?

**Comment by shukaizheng** [comment]
> @wotulong what is configuration for your training? I ran the same code, but I can't reproduce the loss_rec and loss_kl as small as yours. My loss_rec stayed around 0.33 and loss_kl stayed around 0.3.
> Did you make any optimization on the training process?

I don't change anything, It maybe bacause the quality of dataset, I trained it with my own dataset.
